.github/workflows/codeql.yml

```diff
@@ -0,0 +1,84 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
@goagain Do we need this stand alone action if it's already being added to the release pipeline?

Reply:
I am going to move CodeQL to build and test yml.